### ISSUE TITLE
Checklist items go missing

### ISSUE THREAD

**Issue by patricktokeeffe** [dmfs/opentasks#185]
I've been using version 1.1.6 since May or June with whatever the latest version of DAVDroid is, both from the F-droid repo, on Android 5.0.2. The checklist feature is convenient, but the items are not reliably synced. I discover checklist items missing or edits reverted so often that I've developed a nervous 'save' tic. Even then, I end up missing items when I go shopping because they mysteriously vanish at some point.

*constantly = today, twice last week, half-dozen times this month

**Comment by cybafriend** [comment]
I have still the same error. Almost always if I change, delete or add an item to an already existing checklist, after sync the changes are gone.

**Comment by obifur** [comment]
Yes, I have the same problems. I also loose all items when I change a list from local to a davdroid calendar.

CM13, opentasks 1.18.2 , davdroid 1.2

**Comment by korelstar** [comment]
Maybe a problem with your backend? Which software do you use on the server? For me, ownCloud works fine.
